packages/cloud_functions/cloud_functions/lib/src/https_callable.dart

```diff
@@ -29,17 +29,16 @@ class HttpsCallable {
   /// the user is also automatically included.
   Future<HttpsCallableResult> call([dynamic parameters]) async {
     try {
-      final MethodChannel channel = CloudFunctions.channel;
-      final dynamic response = await channel
-          .invokeMethod<dynamic>('CloudFunctions#call', <String, dynamic>{
-        'app': _cloudFunctions._app.name,
-        'region': _cloudFunctions._region,
-        'origin': _cloudFunctions._origin,
-        'timeoutMicroseconds': timeout?.inMicroseconds,
-        'functionName': _functionName,
-        'parameters': parameters,
-      });
-      return HttpsCallableResult._(response);
+      return CloudFunctionsPlatform.instance
```

Comment:
You can either modify this slightly to use `async`/`await` as it is originally or you can get rid of the `async` in the function header.

Reply:
Oh, gosh. Good point.